uint8 controller_mode
float64 goal_x
float64 goal_y
---
uint8 error_code
